package.json

```diff
@@ -4,31 +4,36 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
```

Comment:
run linter and tests as part of prebuild stage and run build with `--no-linter` for quicker builds